doc/source/analyzing/analysis_modules/absorption_spectrum.rst

```diff
@@ -1,12 +1,5 @@
 .. _absorption_spectrum:
 
-.. note::
```

Comment:
Do we want to remove this?

Reply:
The warning notes are repeated on top and below the section header at http://yt-project.org/docs/dev/analyzing/analysis_modules/absorption_spectrum.html and http://yt-project.org/docs/dev/analyzing/analysis_modules/light_ray_generator.html . It seems that they both got there around the same in different commits. Here only the ones on top are removed.

Reply:
Awesome, thank you!